README.md

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 func main() {
-  co := cohere.CreateClient("YOUR_API_KEY")
+  co, err := cohere.CreateClient("YOUR_API_KEY")
```

Comment:
Added

Reply:
the indenting looks a bit weird, can you try autoformatting the code in a Go file then copying it in?

Reply:
I had done that however the indents looked off because the ReadMe's code was space separated instead of tab separated 😞

Reply:
I had done that however the indents looked off because the ReadMe's code was space separated instead of tab separated 😞 
Replaced the tabs with spaces to be symmetric

Reply:
@jimwu6 can you re-review